server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -694,8 +694,8 @@ protected Table getTableWithHeader(final RestRequest request) {
         table.addCell("suggest.total", "sibling:pri;alias:suto,suggestTotal;default:false;text-align:right;desc:number of suggest ops");
         table.addCell("pri.suggest.total", "default:false;text-align:right;desc:number of suggest ops");
 
-        table.addCell("memory.total", "sibling:pri;alias:tm,memoryTotal;default:false;text-align:right;desc:total used memory");
-        table.addCell("pri.memory.total", "default:false;text-align:right;desc:total user memory");
+        table.addCell("memory.total", "sibling:pri;alias:tm,memoryTotal;default:false;text-align:right;desc:total used memory of primaries & replicas");
```

Comment:
@reta Does this seem accurate to you?

Reply:
@kotwanikunal I think the `pri.*` prefix generally reflects primary, so it looks accurate but not complete, we have many other columns which probably have to be documented in a similar fashion, not only `*memory*